File: Leetcode/Array/greatestNumberOfCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        res=[]
        largestNoOfCandies = max(candies)
        for i in range(0,len(candies)):
            if(candies[i]+extraCandies >= largestNoOfCandies):
                res.append(True)
            else:
                res.append(False)
        return res

File: Leetcode/Array/greatestNumberOfCandies.cpp
#include <bits/stdc++.h>  // This will work only for g++ compiler

//Macros for loops
#define for0(i, n) for (int i = 0; i < (int)(n); i++)
#define forr0(i, n) for (int i = (int)(n) - 1; i >= 0; --i) 

using namespace std;
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        
        vector<bool> res;
        int biggestNumberOfCandies = *max_element(candies.begin(), candies.end());
        
        for(int i = 0; i < candies.size(); i++)
        {
            if(candies[i]+extraCandies >= biggestNumberOfCandies){
                res.push_back(true);
            }else
            {
                res.push_back(false);   
            }
        }
        return res;
    }
};
int main(){





return 0;
}

